content/influxdb/cloud-dedicated/query-data/influxql/execute-queries/python.md

```diff
@@ -0,0 +1,290 @@
+    python3 --version
+    ```
+
+    ```sh
+    pip3 --version
+    ```
+
+    If neither `pip` nor `pip<PYTHON_VERSION>` works, follow one of the [Pypa.io Pip installation](https://pip.pypa.io/en/stable/installation/) methods for your system.
+
+### Create a project virtual environment {#venv-install}
+
+1. Create a directory for your Python project and change to the new directory--for example:
+
+    ```sh
+    mkdir ./PROJECT_DIRECTORY && cd $_
+    ```
+
+2. Use the Python `venv` module to create a virtual environment--for example:
+    
+    ```sh
+    python -m venv envs/virtualenv-1
+    ```
+
+   `venv` creates the new virtual environment directory in your project.
+   
+3. To activate the new virtual environment in your terminal, run the `source` command and pass the file path of the virtual environment `activate` script:
+
+    ```sh
+    source envs/VIRTUAL_ENVIRONMENT_NAME/bin/activate
+    ```
+
+    For example:
+    
+    ```sh
+    source envs/virtualenv-1/bin/activate
+    ```
+<!---------------------------------- End venv --------------------------------->
+{{% /tab-content %}}
+{{% tab-content %}}
+<!-------------------------------- Begin conda -------------------------------->
+
+### Install Anaconda {#conda-install}
+
+1. Follow the [Anaconda installation instructions](https://docs.continuum.io/anaconda/install/) for your system.
+2. Check that you can run the `conda` command:
+
+    ```sh
+    conda
+    ```
+
+3. Use `conda` to create a virtual environment--for example:
+
+    ```sh
+    conda create --prefix envs/virtualenv-1 
+    ```
+
+    `conda` creates a virtual environment in a directory named `./envs/virtualenv-1`.
+
+4. To activate the new virtual environment, use the `conda activate` command and pass the directory path of the virtual environment:
+
+    ```sh
+    conda activate envs/VIRTUAL_ENVIRONMENT_NAME
+    ```
+
+    For example:
+
+    ```sh
+    conda activate ./envs/virtualenv-1
+    ```
+
+<!--------------------------------- END conda --------------------------------->
+{{% /tab-content %}}
+    {{< /tabs-wrapper >}}
+
+When a virtual environment is activated, the name displays at the beginning of your terminal command line--for example:
+  
+{{% code-callout "(virtualenv-1)"%}}
+```sh
+(virtualenv-1) $ PROJECT_DIRECTORY
+```
+{{% /code-callout %}}
+
+## Query InfluxDB
+
+1. [Install the influxdb3-python library](#install-the-influxdb3-python-library)
+2. [Create an InfluxDB client](#create-an-influxdb-client)
+3. [Execute a query](#execute-a-query)
+
+### Install the influxdb3-python library
+
+The `influxdb_client_3` module provides a simple and convenient way to interact
+with {{< cloud-name >}} using Python. This module supports both writing data to
+InfluxDB and querying data using SQL or InfluxQL queries.
+
+{{% note %}}
+_To query data with **InfluxQL** and Python, see
+[Use InfluxQL with Python](/influxdb/cloud-dedicated/query-data/influxql/execute-queries/python/)._
+{{% /note %}}
+
```

Comment:
```suggestion
```

Self-referencing? Is it supposed to be SQL?

Reply:
Ah thanks. Copy-pasta.